state-manage-assign/src/components/Table.tsx

```diff
@@ -0,0 +1,260 @@
+import Button from '@mui/material/Button'
+import './Table.css'
+import './Filter';
+
+
+
+function createData(
+  name: string,
+  adjudication: string,
+  status: string,
+  location: string,
+  date: string,
+) {
+  return { name, adjudication, status, location, date };
+}
+
+const rows = [
+  createData('John Smith', '-', 'CLEAR', 'Barrouallie', '2/22/2022'),
+  createData('Serene', '-', 'CLEAR', 'Vanrsborg', '3/13/2022'),
+  createData('Walsh', '-', 'CONSIDER', 'Sukamanah', '7/2/2022'),
+  createData('Maurizia', '-', 'CLEAR', 'Sukamanah', '2/20/2022'),
+  createData('Kendre', '-', 'CLEAR', 'Beutong Ateuh', '5/19/2022'),
+  createData('Erastus', '-', 'CLEAR', 'Hoviyn Am', '5/19/2022'),
+  createData('Jereme', '-', 'CONSIDER', 'Sharingol', '5/19/2022'),
+  createData('John Smith', '-', 'CONSIDER', 'Lianyun', '5/19/2022'),
+  createData('Cari', '-', 'CLEAR', 'Taboao da Serra', '5/19/2022'),
+  createData('Kimble', '-', 'CONSIDER', 'Veseli nad Moravou', '5/19/2022'),
+
+];
+
+
+export default function BasicTable() {
+  const [flag, setFlag] = useState<boolean>(false);
+
+
+  const [searchQuery, setSearchQuery] = React.useState('');
+
+  
+
+  const [open, setOpen] = React.useState(false);
+  const filteredRows = rows.filter((row) => 
+    row.name.toLowerCase().includes(searchQuery.toLowerCase()),
+    
+    
+  );
+  const [filteredData, setFilteredData] = useState(rows);
+  const [stat, setStat] = useState("All");
+  useEffect(() => {
+    if (stat === "All") {
+      setFilteredData(rows);
+      
+    } else if (stat === "CLEAR" || stat === "CONSIDER") {
+      setFilteredData(rows.filter((dt) => dt.status === stat));
+    } else {
+      setFilteredData(rows.filter((dt) => dt.adjudication === stat));
+    }
+    
+  }, [stat]);
+
+  
+  
+  
+  function handleListKeyDown(event: React.KeyboardEvent) {
+    if (event.key === 'Tab') {
+      event.preventDefault();
+      setOpen(false);
+    } else if (event.key === 'Escape') {
+      setOpen(false);
+    }
+  }
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  
+  
+ 
+  return (
+    <div>
+      
+      <div className='header-top'>
+        <Text level={'Candidates'}></Text>
+        <div >
+        
+        <Button className='button' variant="outlined" startIcon={<SystemUpdateAltIcon />} 
+          style={{borderColor: '#696A6E', color : '#696A6E', margin : '15px', textTransform:"none"}}
+          
+          >
+          Export
+        </Button>
+        
+        <Button className='button' variant="contained" startIcon={<AddIcon />}
+          style={{ color: '#FFF',borderColor: 'blue', textTransform:"none"}}>
+          Manual Order
+        </Button>
+        
+        </div>
+        
+    </div>
+    <Paper className='parentCont' style={{borderRadius:'20px'}}>
```

Comment:
for now giving px is fine.
Just checkout how you can use theme.spacing(n) instead of px when using MUI.
Also check how to setup theme file in mui project.
Reference : https://mui.com/material-ui/customization/theming/

Reply:
Done